packages/hub-nodejs/biome.json

```diff
@@ -8,7 +8,7 @@
     "indentSize": 2,
     "indentStyle": "space",
     "lineWidth": 120,
-    "ignore": ["src/generated/*.ts"]
+    "ignore": ["src/generated/*.ts","src/scripts/*.ts"]
```

Comment:
are we leaving this change in long-term? I'd prefer to move away from polluting their projects with our own experimental changes, and move towards a separate project housing our scripts/apps in their monorepo. This can stay as is while we're dropping experimental scripts into `hub-nodejs`, but we should clean this up longer-term.

Reply:
no it's temporary for sure

Reply:
In this case I'd also rather we figure out how to get biome working properly. I'll try VSCode, maybe it'll be better than the Webstorm extension.